README.md

```diff
@@ -194,6 +194,20 @@ Release::Notes.configure do |config|
   # Defaults to `tag`.
   # @return [String]
   config.for_each_ref_format = "tag"
+
+  # Determines whether to use the last two tags to
+  # find commits for the output or if this gem should just
+  # find all commits after previous tag
+  # Defaults to `true`.
+  # @return [Boolean]
+  config.update_release_notes_before_tag = true
+
+  # The newest tag that should be used as the header title
+  # if updating the release notes before the actual tag
+  # is pushed.
+  # Defaults to empty string.
+  # @return [String]
+  config.newest_tag = ""
```

Comment:
Maybe it would be worth adding some info in the readme related to this addition?

Reply:
Doesn't belong in this PR but it would be nice to audit the readme and make sure everything in there is up to date and include info that is important to getting started with the gem that is not already included.